### ISSUE TITLE
Can't build app for tvOS

### ISSUE THREAD

**Issue by Watson1978** [HipByte/motion-game#15]
```
$ gem i motion-game
Successfully installed motion-game-1.1
1 gem installed

$ motion create --template=motion-game HelloWorld
    Create HelloWorld
    Create HelloWorld/app/application.rb
    Create HelloWorld/app/main_scene.rb
    Create HelloWorld/config/android.rb
    Create HelloWorld/config/ios.rb
    Create HelloWorld/config/tvos.rb
    Create HelloWorld/Gemfile
    Create HelloWorld/Rakefile

$ rake tvos:simulator
     Build build/ios/AppleTVSimulator-9.0-Development
  WARNING! Could not find .bridgesupport file for framework "CoreMotion".
rake aborted!
Errno::ENOENT: No such file or directory - /Users/watson/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/motion-game-1.1/lib/motion-game/../../build/tvos
/Library/RubyMotion/lib/motion/project/vendor.rb:47:in `chdir'
/Library/RubyMotion/lib/motion/project/vendor.rb:47:in `build'
/Library/RubyMotion/lib/motion/project/builder.rb:69:in `block in build'
/Library/RubyMotion/lib/motion/project/builder.rb:68:in `each'
/Library/RubyMotion/lib/motion/project/builder.rb:68:in `build'
/Library/RubyMotion/lib/motion/project/app.rb:78:in `build'
/Library/RubyMotion/lib/motion/project/template/tvos.rb:68:in `block (2 levels) in <top (required)>'
/Library/RubyMotion/lib/motion/project/template/tvos.rb:91:in `block in <top (required)>'
Tasks: TOP => build:simulator
(See full trace by running task with --trace)
```

**Comment by Watson1978** [comment]
Looks like motion-game v1.1 does not have pre-build library for tvOS
```
[master][~/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/motion-game-1.1]$ ls
build    doc      lib      template
[master][~/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/motion-game-1.1]$ ls -l build
total 0
drwxr-xr-x   5 watson  staff  170  2 28 04:14 android
drwxr-xr-x  12 watson  staff  408  2 28 04:14 ios
```

**Comment by Watson1978** [comment]
@lrz Please add me to owner of https://rubygems.org/gems/motion-game

**Comment by Watson1978** [comment]
Fixed with motion-game v1.1.1
